gradio/components.py

```diff
@@ -2343,13 +2393,63 @@ def postprocess(self, y):
 
 # Static Components
 class Markdown(Component):
-    def __init__(self, default_value: str = "", *, label: str, **kwargs):
-        super().__init__(label=label, **kwargs)
+    def __init__(
+        self,
+        default_value: str = "",
+        *,
+        label: Optional[str] = None,
+        css: Dict = {},
+        **kwargs,
+    ):
+        super().__init__(label=label, css=css, **kwargs)
+        self.md = MarkdownIt()
+        self.value = self.md.render(default_value)
+        
+    def get_template_context(self):
+        return {
+            "value": self.value,
+            **super().get_template_context()
+        }
 
 
 class Button(Component):
-    def __init__(self, default_value: str = "", *, label: str, **kwargs):
-        super().__init__(label=label, **kwargs)
+    def __init__(
+        self,
+        default_value: str = "",
+        *,
+        label: Optional[str] = None,
+        css: Dict = {},
+        **kwargs,
+    ):
+        super().__init__(label=label, css=css, **kwargs)
+        self.value = default_value
```

Comment:
These changes do not seem to be related to this PR as well, better have these kind of changes in another PR or commit, in the future.